protobuf/smrpc/smrpc.proto

```diff
@@ -10,6 +10,7 @@ option go_package = "github.com/longhorn/types/pkg/generated/smrpc";
 import "google/protobuf/empty.proto";
 
 service ShareManagerService {
+	rpc FilesystemResize(google.protobuf.Empty) returns (google.protobuf.Empty) {}
```

Comment:
No need the new and old sizes?

Reply:
That's correct - the fs utility, either `resize2fs` or `xfs_growfs` recognizes the necessary growth and just does the right thing.

Reply:
Do we need a method to get the current size of the filesystem inside share manager? Like this one is `set` method and the new one is `get` method?

Reply:
A good question.  Shall I write a separate ticket for that?  I'd hate to hold this up for it.

Reply:
Thinking about it again, I think we can skip it for now. We can add them later when we need the new function